hacking/coreconfig.yaml

```diff
@@ -0,0 +1,36 @@
+--- 
```

Comment:
I think a file such as this one can be useful as documentation, so let's not delete it, but I think we should also have a config file which is a minimal, reasonable, not too verbose default configuration.
Proposal: rename this to `settings.yaml.example`, and put together a second `settings.yaml` which only overrides the defaults, if and where necessary. I'd expect this to boil down to just a few essential entries, including
```
controlport
executor
mesosurl
verbose
veryverbose
workflowconfigurationuri
```